src/cascadia/TerminalApp/Command.cpp

```diff
@@ -11,15 +11,33 @@
 #include <LibraryResources.h>
 
 using namespace winrt::TerminalApp;
+using namespace winrt::Windows::Foundation;
 using namespace ::TerminalApp;
 
 static constexpr std::string_view NameKey{ "name" };
 static constexpr std::string_view IconPathKey{ "iconPath" };
 static constexpr std::string_view ActionKey{ "command" };
 static constexpr std::string_view ArgsKey{ "args" };
+static constexpr std::string_view IterateOnKey{ "iterateOn" };
+static constexpr std::string_view CommandsKey{ "commands" };
+
+static constexpr std::string_view IterateOnProfilesValue{ "profiles" };
+
+static constexpr std::wstring_view ProfileName{ L"${profile.name}" };
 
 namespace winrt::TerminalApp::implementation
 {
+    Command::Command()
+    {
+        _nestedCommandsView = winrt::single_threaded_vector<winrt::TerminalApp::Command>();
+        _setAction(nullptr);
+    }
+
+    Collections::IVector<winrt::TerminalApp::Command> Command::NestedCommands()
+    {
+        return _nestedCommandsView;

```

Comment:
unfortunately, this isn't returning a view -- it's returning a mutable collection